testsuite/MathMLToDisplay/Characters/issue898.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>SVG output of space characters</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../header.js"></script>
+    <script type="text/javascript" src="../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+        var texts = document.getElementsByTagName('text');
+        var check1 = (texts.length === 0) ;        
+        var svgs = document.getElementsByTagName('svg');
+        var check2 = true;
+        for (i=0;i<svgs.length;i++){
+          if (svgs[i].innerHTML.indexOf('font-style=""')>-1){
+            check2 = false;
```

Comment:
Since you don't need to test any further once you ht this, you can ether use

    check2 = false; break;

or

    for (var i = 0; check2 && i < svgs.length; i++) {

earlier to stop the loop when the first failure occurs.